latest/index.bs

```diff
@@ -175,7 +175,309 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
+            |                     # `.zattrs` MAY contain "instance_key", which is the key in `obs` that denotes which instance in "region" the row corresponds to. If "instance_key" is not provided, the values from the `obs` `.zattrs` "_index" key is used.
+            │
+            ├── X                 # You MAY add an zarr array `X`.
+            │   │                 # `X` MUST not be a complex type (i.e., MUST be a single type)
+            │   │                 # `X` MAY be chunked as the user desires.
+            │   ├── .zarray
+            │   ├── 0.0
+            │   │   ...
+            │   └── n.m
+            |
+            ├── layers            # You MAY add a `layers` group, which contains dense matrices with the same shape as X.
+            │   │
+            │   ├── .zgroup
+            │   ├── .zattrs       # `.zattrs` MUST contain `"keys"`, which is an array of the names of the subgroups containing a `layer`.
+            │   │
+            │   └── layer_0       # You MAY add a zarr array for each layer
+            |       |             # Each layer array MUST have the same shape as X
+            |       |             # Each layer array SHOULD be chunked the same as X
+            |       ├── .zarray
+            |       |
+            |       ├── 0.0
+            │       │   ...
+            │       └── n.m
+            │        
+            ├── obs               # You MUST add an obs group container. The obs group holds a table of annotations on the rows in X.
+            │   │                 # The rows in obs MUST be index-matched to the rows in X.
+            │   ├── .zgroup
+            │   │                     
+            │   ├── .zattrs       # `.zattrs` MUST contain `"_index"`, which is the name of the column in obs to be used as the index.           
+            │   │                 # `.zattrs` MUST contain `"column-order"`, which is a list of the order of the non-_index columns.
+            │   │                 # `.zattrs` MUST contain `"encoding-type"`, which is set to `"dataframe"` by AnnData.
+            │   │                 # `.zattrs` MUST contain `"encoding-version"`, which is set to `"0.2.0"` by AnnData.
+            │   │      
+            │   └── col_0         # Each column in the obs table is a 1D zarr array. The rows can be chunked as the user desires.
+            │       ├── .zarray   # However, the obs columns SHOULD be chunked in the same way as the rows in X (if present).
+            │       │
+            │       └─ 0
+            ├── var               # You MAY add a var group container. The var group holds a table of annotations on the columns in X.
+            |   │                 # The rows in var MUST be index-matched to the columns in X (if present). 
+            |   |
+            |   ├── .zattrs       # `.zattrs` MUST contain `"_index"`, which is the name of the column in obs to be used as the index.           
```

Comment:
In the example at https://haniffa.cog.sanger.ac.uk/fetal-immune/fetal-liver/visium/0.0.1/visium_1_anndata.zarr/var/.zattrs the "_index" is "SYMBOL".
```
{
    "_index": "SYMBOL",
    "column-order": [
        "ENSEMBL",
        "feature_types",
        "genome",
        "mt"
    ],
    "encoding-type": "dataframe",
    "encoding-version": "0.2.0"
}
``` 

But I don't see that this refers to anything under
https://haniffa.cog.sanger.ac.uk/fetal-immune/fetal-liver/visium/0.0.1/visium_1_anndata.zarr/obs/.zattrs

```
{
    "_index": "_index",
    "column-order": [
        "in_tissue",
        "array_row",
        "array_col",
        "sample",
        "n_genes_by_counts",
        "log1p_n_genes_by_counts",
        "total_counts",
        "log1p_total_counts",
        "pct_counts_in_top_50_genes",
        "pct_counts_in_top_100_genes",
        "pct_counts_in_top_200_genes",
        "pct_counts_in_top_500_genes",
        "mt_frac",
        "img_id",
        "EXP_id",
        "Organ",
        "Fetal_id",
        "SN",
        "Visium_Area_id",
        "Age_PCW",
        "Digestion time",
        "paths",
        "sample_id",
        "_scvi_batch",
        "_scvi_labels",
        "_indices",
        "total_cell_abundance",
        "label_id"
    ],
    "encoding-type": "dataframe",
    "encoding-version": "0.2.0"
```

Also, this line is a duplicate of the same line under 'obs' above (line 238). Is that correct?

Reply:
It is probably a typo

```diff
- # `.zattrs` MUST contain `"_index"`, which is the name of the column in obs to be used as the index
+ # `.zattrs` MUST contain `"_index"`, which is the name of the column in var to be used as the index
```

In the example, the data for the var index column name specified in `_index` of `var/.zattrs` column is located under `var/{_index}/` (see https://haniffa.cog.sanger.ac.uk/fetal-immune/fetal-liver/visium/0.0.1/visium_1_anndata.zarr/var/SYMBOL/.zattrs).

https://github.com/vitessce/vitessce/blob/10f6f3b/packages/file-types/zarr/src/AnnDataSource.js#L247

Reply:
Column names loaded correctly now: https://deploy-preview-20--ome-ngff-validator.netlify.app/?source=https://haniffa.cog.sanger.ac.uk/fetal-immune/fetal-spleen/visium/0.0.1/visium_1_anndata.zarr

Reply:
So this line still needs to be fixed to `...the column in var to...`